src/Umbraco.Web.UI.Client/src/views/components/umb-node-preview.html

```diff
@@ -1,85 +1,75 @@
 <div class="umb-node-preview" ng-class="{'umb-node-preview--sortable': sortable, 'umb-node-preview--unpublished': published === false }">
     <div class="flex"> <!-- div keeps icon and nodename from wrapping -->
-        <i ng-if="icon" class="umb-node-preview__icon {{ icon }}" aria-hidden="true"></i>
+        <umb-icon ng-if="icon" icon="{{icon}}" class="umb-node-preview__icon {{icon}}"></umb-icon>
```

Comment:
The bound `{{icon}}` property in the class attribute can be removed too, it's not required.